apps/src/templates/EligibilityConfirmDialog.jsx

```diff
@@ -0,0 +1,98 @@
+/** @file Confirm Dialog for Maker Discount Codes */
+import React, {Component} from 'react';
+import i18n from "@cdo/locale";
+import BaseDialog from "./BaseDialog";
+import DialogFooter from "./teacherDashboard/DialogFooter";
+import Button from "./Button";
+
+const styles = {
+  subtitle: {
+    fontSize: 16,
+  }
+};
+
+export default class EligibilityConfirmDialog extends Component {
+  state = {
+    signature: "",
+    activeButton: false,
+  };
+
+  verifyResponse = () => {
+    if (document.getElementById('verify1').checked &&
+        document.getElementById('verify2').checked &&
+        document.getElementById('verify3').checked &&
```

Comment:
We should always avoiding using `document.getElementById` in React components, as it immediately makes our component less modular (now if another component uses `verify1` we might end up accessing that instead).

React has a solution for this called refs https://reactjs.org/docs/refs-and-the-dom.html that we should probably use here. LMK if it's not clear from the article how to use them, and I can help walk you through it.